### DISCUSSION TITLE
New version of Apprise?

### DISCUSSION THREAD

**Discussion by opsnlops** [mcguirepr89/BirdNET-Pi#660] [Q&A] [answered]
Hello!

I'm trying to get my BirdNet-Pi Twitter bot moved to Mastodon after the events of the last few weeks.

Apprise recently gained support for Mastodon (version 1.2), but BirdNet-Pi is still using 1.1. Is there a plan to upgrade this dependency soon?

April

**Comment by ehpersonal38** [comment, marked as answer]
That's certainly something to consider - though at this time I'm not sure the best way to implement automatically upgrading Python dependencies.

In the meantime you can manually update Apprise with this command:

`~/BirdNET-Pi/birdnet/bin/python3 -m pip install -U apprise`

**Reply by opsnlops** [reply]
Ah, yeah, that did work. :)

https://botsin.space/@SouthWhidbeyBirds

Getting on version 1.20 (the most recent) added in Mastodon support, and BirdNet-Pi works great with it!
